wordpress-mobile/WordPress-iOS
# Add "Set your site title" action to the quick start list

Fixes #14362

To test:

1. Create a new site, then dismiss the initial preview
2. Tap on "Yes, Help Me" when prompted
3. Verify that the first element of the quick start list is: "Set your site title"
4. Verify that the icon is Gridicon: pencil
5. Verify that the description matches "Give your site a name that reflects its personality and topic. First impressions count!"
6. Tap on the "Set your site title" action
7. Verify that the spotlight shows on the site title label, and that the notice at the bottom appears, with the text "Select [Your Site Title] to set a new one [Your Site Title] = the current site title
8. Verify that tapping the site title label shows the title setting view, and the updated title is reflected in My Site and My Sites

PR submission checklist:
